test/python/importer/jit_ir/node_import/prim.py

```diff
@@ -162,3 +162,34 @@ def prim_max(x: int):
 
 mb.module.operation.print()
 print()
+
+# CHECK-LABEL:   func.func @__torch__.prim_Constant_scalar() -> !torch.number {
+# CHECK:           %[[A:.*]] = torch.tensor.literal
+# CHECK:           %[[RET:.*]] = torch.aten.ScalarImplicit
+# CHECK:           return %[[RET]] : !torch.number
+import_options = ImportOptions()
+import_options.assumeTensorsHaveValueSemantics = False
+mb.import_function(create_script_function("__torch__.prim_Constant_scalar", """
+graph():
+  %0 : Long(requires_grad=0, device=cpu) = prim::Constant[value={1}]()
+  %1 : Scalar = aten::ScalarImplicit(%0)
```

Comment:
are these tests still relevant?

Reply:
Yes as they test node_import code https://github.com/llvm/torch-mlir/pull/1412/files#diff-05424e18718175ef3b778a9eb98fe120702705e14d57eab13674ebf84df8e28bR201
And test/python/importer/jit_ir/ivalue_import/tensors-value-semantics.py test ivalue_importer.